lib/entities.js

```diff
@@ -18,260 +18,2145 @@ exports.XML_ENTITIES = freeze({
 });
 
 /**
- * A map of currently 241 entities that are detected in an HTML document.
+ * A map of all entities that are detected in an HTML document.
  * They contain all entries from `XML_ENTITIES`.
  *
  * @see XML_ENTITIES
  * @see DOMParser.parseFromString
  * @see DOMImplementation.prototype.createHTMLDocument
  * @see https://html.spec.whatwg.org/#named-character-references WHATWG HTML(5) Spec
+ * @see https://html.spec.whatwg.org/entities.json JSON
```

Comment:
Do you know how often this file changes?
Maybe it is in some version control where this information could be looked up?
(Not required for landing this PR, but I'm wondering whether it would make sense to run a scheduled action that checks whether our version is still in sync and maybe even creates a branch/PR with an update if there is one.)

Reply:
https://html.spec.whatwg.org/multipage/named-characters.html#named-character-references states that it stable and will not change